lines_in_file.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+lines_in_file()
+{
+ cat $1 | wc -l
+}
+
+if [ $# -eq 0 ]
+then
+ echo "no argument is passed"
```

Comment:
You need to specify if your script run successfully or not.  You need to add 'exit 0', 'exit 1' statements